src/EnergyPlus/SingleDuct.cc

```diff
@@ -6275,6 +6275,11 @@ void GetATMixers(EnergyPlusData &state)
                                                                               state.dataSingleDuct->SysATMixer(ATMixerNum).ZoneNum,
                                                                               false,
                                                                               false);
+                            state.dataSingleDuct->SysATMixer(ATMixerNum).ADEff =
+                                std::min(state.dataSize->ZoneSizingInput(SizingInputNum).ZoneADEffCooling,
+                                         state.dataSize->ZoneSizingInput(SizingInputNum).ZoneADEffHeating);
+                            state.dataSingleDuct->SysATMixer(ATMixerNum).DesignPrimaryAirVolRate /=
+                                state.dataSingleDuct->SysATMixer(ATMixerNum).ADEff;
```

Comment:
This is a one time init in GetATMixers that adjusts the ATMixer primary flow based on zone distribution effectiveness. This is all good except without also adjusting the zone OA calculated during the simulation this one time init only affects the size, not the operating flow rate.